hello-pepr-alias/capabilities/alias.e2e.test.ts

```diff
@@ -2,11 +2,12 @@ import { beforeAll, afterAll, describe, it, expect } from "@jest/globals";
 import { TestRunCfg } from "helpers/src/TestRunCfg";
 import { mins, secs, timed } from "helpers/src/time";
 import { fullCreate } from "helpers/src/general";
-import { moduleUp, moduleDown, logs } from "helpers/src/pepr";
+import { moduleUp, moduleDown, logs, untilLogged } from "helpers/src/pepr";
 import { clean } from "helpers/src/cluster";
 import { K8s, kind } from "pepr";
 
 const trc = new TestRunCfg(__filename);
+const SECONDS = 1000;
```

Comment:
```suggestion
```
This doesn't appear to be used anywhere.  Also, there's already the `secs()` helper imported that can be used to convert seconds to ms (should find you need it in the future).